sentry-contexts/src/integration.rs

```diff
@@ -36,6 +33,23 @@ impl ContextIntegration {
     pub fn new() -> Self {
         Self::default()
     }
+
+    /// Add `os` context, enabled by default.
+    pub fn add_os(mut self, add_os: bool) -> Self {
+        self.add_os = add_os;
+        self
+    }
+    /// Add `rust` context, enabled by default.
```

Comment:
_nit_: Add an empty line. 

The `add_` prefixes seem a bit redundant, but since they are speaking they are fine.